ts/packages/node/src/stores/SQLite.ts

```diff
@@ -1,7 +1,9 @@
 import { type SqliteConnectionOptions } from 'typeorm/driver/sqlite/SqliteConnectionOptions';
 import { ORMConfigStore } from './ORM';
 
-export type SQLiteConfigStoreConfiguration = Omit<SqliteConnectionOptions, 'type'>;
+export type SQLiteConfigStoreConfiguration = Omit<SqliteConnectionOptions, 'type'> & {
+  tableName?: string;
+};
```

Comment:
I feel like modifying `ORMConfigStore` to accept a table only for `SQLiteConfigStore` is quirky. I feel like we should make this new property available for all stores extending `ORMConfigStore`. Let's get a product opinion on this.

Reply:
All ORM config stores should have the new table name props

Reply:
Done